src/main/java/core/basesyntax/service/CsvFileWriter.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+
+public interface CsvFileWriter {
+    void reportFile(Map<String, Long> balance, String filePath);
```

Comment:
I'm not sure, maybe you should use Fruit instead String?

Reply:
I think it's not a mistake in this task

Reply:
It's better to use Fruit in case you will add some fields to it but you may leave it